platform/darwin/src/MGLVectorTileSource_Private.h

```diff
@@ -6,13 +6,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, readonly, getter=isMapboxStreets) BOOL mapboxStreets;
 
-+ (NSSet<NSString *> *)mapboxStreetsLanguages;
++ (NS_SET_OF(NSString *) *)mapboxStreetsLanguages;
```

Comment:
These macros (like `NS_SET_OF()`) no longer exist in `master` (as of #11776) so this change needs to be reverted.